Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.py

```diff
@@ -508,21 +516,25 @@ def main() -> None:
     :return:
     :rtype:
     """
-
+    params = demisto.params()
     # if your Client class inherits from BaseClient, SSL verification is
     # handled out of the box by it, just pass ``verify_certificate`` to
     # the Client constructor
-    verify_certificate = not demisto.params().get('insecure', False)
+    verify_certificate = not params.get('insecure', False)
 
     # if your Client class inherits from BaseClient, system proxy is handled
     # out of the box by it, just pass ``proxy`` to the Client constructor
-    proxy = demisto.params().get('proxy', False)
-    app_id = demisto.params().get('app_id')
-    base_url = demisto.params().get('base_url')
+    proxy = params.get('proxy', False)
+    app_id = params.get('app_id')
+    base_url = params.get('base_url')
+
+    tenant_id = params.get('tenant_id')
+    self_deployed = params.get('self_deployed', False)
```

Comment:
```suggestion
    self_deployed = argToBoolean(params.get('self_deployed', 'false'))
```

Reply:
@bakatzir 
I used `type: 8` which is a radio button. the values are booleans `True/False` and not the string representation of it. 
Keeping as is.